docs/developer-guide/building-and-developing.md

```diff
@@ -26,6 +26,31 @@ Then point your preferred browser to [http://localhost:8081](http://localhost:80
 
 The HomePage contains links to the available demo applications.
 
+### Building the documentation
+
+MapStore2 uses JSDoc to annotate the components, so the documentation can be automatically generated using [docma](http://onury.github.io/docma/).  
+Please see http://usejsdoc.org/ for further information about code documentation.  
```

Comment:
Ask to follow the existing examples of actions, reducers, components, epics and plugins to understand how to document a component